ivre/db/mongo.py

```diff
@@ -1163,6 +1209,27 @@ def searchcategory(cat, neg=False):
                 return {'categories': {'$in': cat}}
         return {'categories': cat}
 
+    def searchlabel(self, group, label=None, regexp=False, neg=False):
```

Comment:
This is not the expected API. I should provide either a regular expression, or a string, and the method should figure out what to do with it. See other methods, such as `.searchcategory()`.